vendor/github.com/Venafi/vcert/v4/Makefile

```diff
@@ -43,20 +43,32 @@ gofmt:
 	! gofmt -l . | grep -v ^vendor/ | grep .
 
 test: get linter
-	go test -v -cover .
-	go test -v -cover ./pkg/certificate
-	go test -v -cover ./pkg/endpoint
-	go test -v -cover ./pkg/venafi/fake
-	go test -v -cover ./cmd/vcert
+	go test -v -coverprofile=cov1.out .
+	go tool cover -func=cov1.out
+	go test -v -coverprofile=cov2.out ./pkg/certificate
```

Comment:
Are we sure what are we doing here? None of the following paths exist on this project:
/pkg/certificate
/pkg/endpoint
/pkg/venafi/fake
/pkg/policy
/pkg/util
/cmd/vcert

Reply:
Actually, these is because of the vendor...